src/views/conference/2018/details/details.jsx

```diff
@@ -0,0 +1,114 @@
+const classNames = require('classnames');
+const connect = require('react-redux').connect;
+const PropTypes = require('prop-types');
+const React = require('react');
+const render = require('../../../../lib/render.jsx');
+
+const detailsActions = require('../../../../redux/conference-details.js');
+
+const Page = require('../../../../components/page/conference/2018/page.jsx');
+
+require('./details.scss');
+
+class ConferenceDetails extends React.Component {
+    componentDidMount () {
+        let pathname = window.location.pathname.toLowerCase();
+        if (pathname[pathname.length - 1] === '/') {
+            pathname = pathname.substring(0, pathname.length - 1);
+        }
+        const path = pathname.split('/');
+        const detailsId = path[path.length - 2];
```

Comment:
Just a suggestion (in the future?): `window.location.pathname.toLowerCase().split('/').filter(Boolean)[2]`